docs/use-the-network/coverage-mapping/coverage-mapping.mdx

```diff
@@ -160,7 +160,7 @@ Dragino LGT-92, but the steps are effectively the same for any mapping device.
 - RAK Wireless RAK7200
   - [Product page](https://store.rakwireless.com/products/rak7200-lpwan-tracker?variant=29669491867693)
 
-### DIY
+### DIY mappers
```

Comment:
```suggestion
### DIY Mappers
```

Reply:
If changing this title structure here, it should also be mirrored on 'Plug & Play' and other titles.